rpm/authselect.spec.in

```diff
@@ -35,6 +35,16 @@ supported by authselect.
 
 %package libs
 Summary: Utility library used by the authselect tool
+# Required by scriptlets
+Requires: %{_bindir}/awk
+Requires: %{_bindir}/cp
+Requires: %{_bindir}/echo
+Requires: %{_bindir}/find
+Requires: %{_bindir}/grep
+Requires: %{_bindir}/rm
+Requires: %{_bindir}/sed
+Requires: %{_bindir}/systemctl
```

Comment:
I think this is incorrect, shouldn't you require systemd? Also, I can't imagine a system where systemctl does not exist, is this even necessary?